xml/upgrade-3.10-1.xsl

```diff
@@ -51,6 +53,12 @@
 
 <!-- Cluster properties -->
 
+<!-- Rename crmd-integration-timeout property to join-integration-timeout -->
```

Comment:
Property renames should happen *before* removing duplicates. I'll correct that before the next push.